ca/certificates.go

```diff
@@ -444,7 +444,10 @@ func NewRootCA(rootCertBytes, signCertBytes, signKeyBytes []byte, certExpiry tim
 		pool.AddCert(cert)
 	}
 
-	// Calculate the digest for our Root CA bundle
+	// Make sure the root certs are normalized before getting the digest, so that we can generate a consistent digest
+	// even if the root certs are.  Also store the normalized root cert in the RootCA object.
+	rootCertBytes = NormalizePEMs(rootCertBytes)
```

Comment:
I'm not sure what it means to do the normalization when comparing bytes. I was thinking we could just normalize when we generate a certificate or when the user provides a new certificate, but avoid touching anything that's already in place.